CHANGELOG.md

```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to API callable by dapps
+- [#11849](https://github.com/MetaMask/metamask-extension/pull/11849): Add subtitles for additional languages (Secret Recovery Phrase Video)
+- [#11772](https://github.com/MetaMask/metamask-extension/pull/11772): Add tooltip to better explain possible effects of setting gas fees below MetaMask's estimates
+
+### Changed
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
+- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Autofilling new issue link with url and domain (Phishing detect page)
+- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disabled spell-check on ens input
+- [#12013](https://github.com/MetaMask/metamask-extension/pull/12013): Improving warning text for eth_sign transactions
+- [#11945](https://github.com/MetaMask/metamask-extension/pull/11945): Send user to account page upon adding a custom network
+- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
+- [#11796](https://github.com/MetaMask/metamask-extension/pull/11796): Update Max Fee label in the edit gas display
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Wrapping and unwrapping native currencies (e.g. ETH -> WETH) through Swaps now uses the wrapping contracts directly
+- [#12056](https://github.com/MetaMask/metamask-extension/pull/12056): Updated labels and behaviour of duplicate cancel buttons on the send screen when editing a transaction
+
+### Fixed
+- [#12140](https://github.com/MetaMask/metamask-extension/pull/12140): Fixed max priority fee field so that there are not errors when attempting to enter more than 15 significant digits
+- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fixed erroneous ERC-20 Token amount on the confirmation screen
```

Comment:
```suggestion
- [#12169](https://github.com/MetaMask/metamask-extension/pull/12169): Fix handling of erroneous ERC-20 Token amount on the confirmation screen
  - Previously, if you entered a token amount with too many decimal places, it would corrupt the token amount in the transaction. Now we truncate any decimals beyond what the token allows instead.
```